cloud/azure/database/mariadb/mode/queries.pm

```diff
@@ -0,0 +1,141 @@
+#
+# Copyright 2022 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
```

Comment:
I can't find the metric in the official documentation, did you test it again a MariaDB Azure deployment? 
ref: https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported#microsoftdbformariadbservers

Reply:
Completly removed.